File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.js
/**
 * @param {number} n
 * @return {number[]}
 */
var sumZero = function(n) {
  const ans = []
  for (let i = 1; i <= ~~(n/2); i++) {
    ans.push(-i, i)
  }
  if (n%2) ans.push(0)
  return ans
};

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.go
func sumZero(n int) []int {
  ans := make([]int, n)
  for i := 0; i < n/2; i++ {
    ans[i] = -i-1
    ans[n-1-i] = i+1
  }
  if n % 2 == 1 {
    ans[n/2] = 0
  }
  return ans
}